test/unit/Chainweb/Test/Pact5/CutFixture.hs

```diff
@@ -94,17 +100,25 @@ data Fixture = Fixture
     }
 makeLenses ''Fixture
 
+class HasFixture a where
```

Comment:
I think this kind of abstraction makes some sense, since Fixtures can often be nested within one another.